### DISCUSSION TITLE
Gluco Check in Norwegian?

### DISCUSSION THREAD

**Discussion by SanderRemote20** [nielsmaerten/gluco-check#165] [General]
I've started on the Norwegian translation.
I'm guessing that this won't work until translations are completed?

**Comment by nielsmaerten** [comment]
Hi, thanks for translating!

Yes, to make a new translation of gluco check available to **_everyone_**, it needs to be 100% translated.
But... if you'd like to test the app **_just for yourself_** before releasing to everyone else, then only "assistant-action" and "assistant-responses" are needed.

I see you've already translated "assistant-responses". If you translate "assistant-action" as well and give me a heads up when you're done, I can set you up with a test version of Norwegian Guco Check

**Reply by SanderRemote20** [reply]
I think I'm done with the translating now :-) Would love to test the app!

**Comment by nielsmaerten** [comment]
Awesome, thanks for the help!
I'll get started setting up the Norwegian test version.

I just need to know which email address to add.
Can you send me an email at beta@glucocheck.app from the address you'd like
to use?

**Reply by SanderRemote20** [reply]
Happy to help 😊 I've sent you an email 👍
